CFAST.sh

```diff
@@ -0,0 +1 @@
+java -jar ./server/engine/target/server.jar cfast -ws $1
```

Comment:
I think this file belongs to CCF repository.

Reply:
Ok